frontend/src/components/homepage/StoryCard.tsx

```diff
@@ -58,6 +59,7 @@ const StoryCard = ({
   youtubeLink,
   level,
   language,
+  displayMyStories,
```

Comment:
nitpick: is there another name that we could use? `displayMyStories` feels like a homepage-level logic concept, that seems distinct from what the `storyCard` component should know
- maybe `isAssignedToUser`, `storyInProgress`?

Reply:
What do you think of `isMyStory`?

Reply:
Yep, don't mind that at all